android/src/main/java/com/mapbox/mapboxgl/StaticMethodHandler.java

```diff
@@ -0,0 +1,80 @@
+package com.mapbox.mapboxgl;
+
+import android.content.Context;
+import android.util.Log;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.PluginRegistry;
+
+class StaticMethodHandler implements MethodChannel.MethodCallHandler {
```

Comment:
Can this class be renamed a more useful name? StaticMethodHandler doesn't tell the reader what this class is responsible for.

Reply:
Sure, would `GlobalMethodHandler` work better?  I went with `Static` because its not tied to a Map instance.

Reply:
Renamed to `GlobalMethodHandler`